ingestion/tests/cli_e2e/test_cli_dbt_redshift.py

```diff
@@ -66,7 +66,7 @@ def assert_for_vanilla_ingestion(
     ) -> None:
         self.assertTrue(len(source_status.failures) == 0)
         self.assertTrue(len(source_status.warnings) == 0)
-        self.assertTrue(len(source_status.filtered) == 5)
+        self.assertTrue(len(source_status.filtered) == 8)
```

Comment:
@OnkarVO7 do we know why this went from 5 to 8?

Reply:
Yes I created a few schemas in our redshift instance which got filtered here so the filtered count increased